security/security.md

```diff
@@ -138,10 +138,17 @@ To backup a database through phpMyAdmin, select the database in phpMyAdmin by cl
 
 #### WordPress uses security keys
 
-The wp-config.php file uses security keys (AUTH_KEY, SECURE_AUTH_KEY, LOGGED_IN_KEY, and NONCE_KEY) to ensure better encryption of information stored in the user's cookies. A good security key should be long (60 characters or longer), random and complicated enough. Hosters should verify that the security keys are set up and that they at least contain alphabetic and numeric characters.
+The `wp-config.php` file uses security keys (`AUTH_KEY`, `SECURE_AUTH_KEY`, `LOGGED_IN_KEY`, and `NONCE_KEY`) to ensure better encryption of information stored in the user's cookies. As a best practice security keys should be 60 characters or longer, random and complicated, containing both alphabetic and numeric characters.
 
-**WordPress Security Key Generator https://api.wordpress.org/secret-key/1.1/salt/ can be used to change your security keys.**
+**[The WordPress Security Key Generator](https://api.wordpress.org/secret-key/1.1/salt/) should be used to generate security keys.**
```

Comment:
Since getting these keys from WP.org makes the install process/security rely on an outside source for keys, is this a good security recommendation?

Reply:
What do you think about adding a note that `WordPress.org` is a trusted source but folks should be wary about non-trusted sources?

Reply:
I think the main issue with the suggestion is that it relies on a single outside source (in this case, WP.org), so if it's compromised, this would be a problem for new installs.

Certainly, using this rather than the default is still better.